### ISSUE TITLE
Where is the error()?

### ISSUE THREAD

**Issue by xrdavies** [taosdata/TDengine#958]
When I tried to build Alpine version I met such code that cannot find where is the error() coming from.

```
void shellParseArgument(int argc, char *argv[], struct arguments *arguments) {
  char verType[32] = {0};
  #ifdef CLUSTER
    sprintf(verType, "enterprise version: %s\n", version);
  #else
    sprintf(verType, "community version: %s\n", version);
  #endif
  
  argp_program_version = verType;
  
  argp_parse(&argp, argc, argv, 0, 0, arguments);
  if (arguments->abort) {
    error(10, 0, "ABORTED");
  }
}
```

Could you point out where can I find the definition of `error(10, 0, "ABORTED")` ?

**Comment by xrdavies** [comment]
#75 related.

**Comment by hzcheng** [comment]
It should be in <error.h>. If you cannot compile, you can just abort();

**Comment by xrdavies** [comment]
> It should be in <error.h>. If you cannot compile, you can just abort();

Sorry, in which lib error.h is contained? Never seen such header file before. Anyway, let me remove it.

**Comment by xrdavies** [comment]
@hzcheng Hi, which branch should I submit PR?

**Comment by xrdavies** [comment]
Fixed in #971
